shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -0,0 +1,20 @@
+# coding: utf-8
+
+# TODO: check in unit test if dcm2bids_scaffold is installed, and also check for the required version.
+
+import subprocess
+
+from shimmingtoolbox import __dir_config_dcm2bids__
+
+
+def dicom_to_nifti(path_dicom, path_nifti, subject_id='sub-01', path_config_dcm2bids=__dir_config_dcm2bids__):
+    """ Converts dicom files into nifti files by calling dcm2bids
+
+    Args:
+        path_dicom (str): path to the input dicom folder
+        path_nifti (str): path to the output nifti folder
+
+    """
+    cmd = 'dcm2bids -d {} -p {} -o {} -l DEBUG -c {}'.format(
+        path_dicom, subject_id, path_nifti, path_config_dcm2bids)
+    subprocess.run(cmd.split(' '))
```

Comment:
Mixing string manipulation and execution is prone to [code injection problems](https://owasp.org/www-community/attacks/Command_Injection). Luckily, there's a safe way:
 
```suggestion
    subprocess.run(['dcm2bids', '-d', path_dicom, '-p', subject_id, '-o', path_nifti, '-l', 'DEBUG', '-c', path_config_dcm2bids])
```

Reply:
hum, so i guess you're not a big fan of [this addition](https://github.com/shimming-toolbox/shimming-toolbox-py/pull/15/files#diff-47cba49b13bb4071b92cf726a6c0bc5cR11-R18) in @po09i branch then. We should all agree on good practice to have harmonized code across PRs